drivers/gles3/shaders/scene.glsl

```diff
@@ -1523,6 +1523,9 @@ void main() {
 #CODE : FRAGMENT
 	}
 
+	// Keep albedo values in positive number range as negative values "wraparound" into positive numbers resulting in wrong colors
+	albedo = max(albedo, 0.0);
```

Comment:
```suggestion
	albedo = max(albedo, vec3(0.0));
```
This should be a vec3 as many drivers don't allow implicitly converting between a float and a vec3